client/app/instructor/verifystudents/verifystudents.controller.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+angular.module('venueApp')
+  .controller('VerifystudentsCtrl', function ($scope, Auth, User, Section) {
+    Auth.getCurrentUser((user) => {
+      $scope.user = user;
+      User.getFullSections({id:$scope.user._id})
```

Comment:
this will work with current dev but this method is about to be removed for getSections